kvs-key-value-pairs/README.md

```diff
@@ -0,0 +1,9 @@
+## Use key-value pairs
+
+**CloudFront Functions event type: viewer request**
+
+The example uses key-value pairs from an Amazon CloudFront [key value store](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/kvs-with-functions.html) in a CloudFront function. 
+
+The example shows a function that uses the content of the URL in the HTTP request to look up a custom path in the key value store. CloudFront then uses that custom path to make the request. This function helps manage the multiple paths that are part of a website. 
```

Comment:
I would add an example here. For example, To update the version of a blog platform on a website. The old blog has origin path `/blog-v1` while the new blog has origin path `/blog-v2`. This function can lookup the URL path of the incoming request and rewrite the URL path (`/blog-v1`) to the new version of the blog (`/blog-v2`).